Dima/text/count_words_in_a_string/test_count_of_words.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+from Dima.text.count_words_in_a_string.count_of_words import count_of_words_input
+from Dima.text.count_words_in_a_string.count_of_words import count_of_words_str
+from Dima.text.count_words_in_a_string.count_of_words import count_of_words_text
+
+
+class CountWordsTest(unittest.TestCase):
+
+    def test_count_words_with_none_value(self):
```

Comment:
Please download txt file for 'War and Peas' by Tolstoy and enter it as a test value!

Reply:
566317 with empty strings

Reply:
Please add it as a test. With asserts